recipes/libsigcpp/2.10.x/conanfile.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

Reply:
Why should I remove this? https://github.com/conan-io/conan-center-index/blob/master/recipes/libjpeg/all/conanfile.py#L1-L2
Here it is present